### ISSUE TITLE
Enable/Disable salemaker by clicking red/green box doesn't work

### ISSUE THREAD

**Issue by scottcwilson** [zencart/zencart#6568]
As reported in this thread: 
https://www.zen-cart.com/showthread.php?230239-SaleMaker-status-icons-not-enabling-disabling-sales

**Comment by piloujp** [comment]
The form that toggle status does not have a submit button (in admin/salemaker.php):
After line 478...
```
                  <td  class="dataTableContent text-center">
                      <?php
                      if ($salemaker_sale['sale_status'] == '1') {
                        echo zen_draw_form('setflag_products', FILENAME_SALEMAKER, 'action=setflag&sID=' . $salemaker_sale['sale_id'] . (isset($_GET['page']) ? '&page=' . $_GET['page'] : '') . (isset($_GET['search']) ? '&search=' . $_GET['search'] : ''));
                        ?>
                      <?php echo zen_icon('enabled', IMAGE_ICON_STATUS_ON, 'lg', true); ?>
                      <input type="hidden" name="flag" value="0">
                      <?php echo '</form>'; ?>
                      <?php
                    } else {
                      echo zen_draw_form('setflag_products', FILENAME_SALEMAKER, 'action=setflag&sID=' . $salemaker_sale['sale_id'] . (isset($_GET['page']) ? '&page=' . $_GET['page'] : '') . (isset($_GET['search']) ? '&search=' . $_GET['search'] : ''));
                      ?>
                      <?php echo zen_icon('disabled', IMAGE_ICON_STATUS_OFF, 'lg', true); ?>
                      <input type="hidden" name="flag" value="1">
                      <?php echo '</form>'; ?>
                      <?php
                    }
                    ?>
                  </td>
```
I I add some button like this it works:
```
                  <td  class="dataTableContent text-center">
                      <?php
                      if ($salemaker_sale['sale_status'] == '1') {
                        echo zen_draw_form('setflag_products', FILENAME_SALEMAKER, 'action=setflag&sID=' . $salemaker_sale['sale_id'] . (isset($_GET['page']) ? '&page=' . $_GET['page'] : '') . (isset($_GET['search']) ? '&search=' . $_GET['search'] : ''));
                        ?>
                  <button type="submit" class="btn btn-status">
                      <?php echo zen_icon('enabled', IMAGE_ICON_STATUS_ON, 'lg', true); ?>
                  </button>
                      <input type="hidden" name="flag" value="0">
                      <?php echo '</form>'; ?>
                      <?php
                    } else {
                      echo zen_draw_form('setflag_products', FILENAME_SALEMAKER, 'action=setflag&sID=' . $salemaker_sale['sale_id'] . (isset($_GET['page']) ? '&page=' . $_GET['page'] : '') . (isset($_GET['search']) ? '&search=' . $_GET['search'] : ''));
                      ?>
                  <button type="submit" class="btn btn-status">
                      <?php echo zen_icon('disabled', IMAGE_ICON_STATUS_OFF, 'lg', true); ?>
                  </button>
                      <input type="hidden" name="flag" value="1">
                      <?php echo '</form>'; ?>
                      <?php
                    }
                    ?>
                  </td>
```

**Comment by piloujp** [comment]
With a little bit of optimization, it is better:
```
                  <td  class="dataTableContent text-center">
                      <?php echo zen_draw_form('setflag_products', FILENAME_SALEMAKER, 'action=setflag&sID=' . $salemaker_sale['sale_id'] . (isset($_GET['page']) ? '&page=' . $_GET['page'] : '') . (isset($_GET['search']) ? '&search=' . $_GET['search'] : ''));
                        ?>
                  <button type="submit" class="btn btn-status">
                      <?php
                      if ($salemaker_sale['sale_status'] == '1') {
                        echo zen_icon('enabled', IMAGE_ICON_STATUS_ON, 'lg', true) . "
                      </button>
                      " . '<input type="hidden" name="flag" value="0">' . "\n";
                    } else {
                        echo zen_icon('disabled', IMAGE_ICON_STATUS_OFF, 'lg', true) . "\n
                      </button>\n
                      " . '<input type="hidden" name="flag" value="1">' . "\n";
                    }
                    ?>
                  </form>
                  </td>
```

**Comment by scottcwilson** [comment]
Would you please PR?

**Comment by scottcwilson** [comment]
Fixed by #6585.
